.github/workflows/tests.yml

```
@@ -17,22 +17,28 @@ jobs:
     - uses: actions/checkout@v2
 
     - name: Setup Conda
-      uses: s-weigand/setup-conda@v1
+      #uses: s-weigand/setup-conda@v1.0.5
+      uses: s-weigand/setup-conda@fix-env-activation
```

```suggestion
      uses: s-weigand/setup-conda@fix-env-activation-dist
```
If this fixes the issue with the activation of the `TEST` env.
I will release a new version of `setup-conda` which should fix the usage of the `v1` tag.
Thanks for testing ❤️ 
Ref:
https://github.com/s-weigand/setup-conda/issues/125

---

Sorry for the missing `-dist` there. I tried that using the mobile web interface during the weekend, it was doomed to fail ;-p

Thanks for the quick fix. Let's see how the CIs go.

---

@s-weigand  works like a charm! Thanks!! I'll wait for the release to merge this one.